config.yaml

```diff
@@ -24,8 +24,7 @@ additional_arches: [aarch64, ppc64le, s390x]
 source_config:
   url: https://github.com/coreos/fedora-coreos-config
 
```

Comment:
should we add `see bucket layout in https://github.com/coreos/fedora-coreos-tracker/issues/189` in a comment here?